docker/Linux-JDK17/Dockerfile

```diff
@@ -0,0 +1,56 @@
+FROM ubuntu:lunar
+
+ARG DEBIAN_FRONTEND=noninteractive
+
+# Install various dependencies:
+# * ca-certificates is needed by wget
+# * ffmpeg is needed by FfmpegProcessor
+# * wget download stuffs in this dockerfile
+# * libopenjp2-tools is needed by OpenJpegProcessor
+# * All the rest is needed by GrokProcessor
+RUN apt-get update && apt-get install -y --no-install-recommends \
+  ca-certificates \
+  ffmpeg \
+  wget \
+  libopenjp2-tools \
+  liblcms2-dev \
+  libpng-dev \
+  libzstd-dev \
+  libtiff-dev \
+  libjpeg-dev \
+  zlib1g-dev \
+  libwebp-dev \
+  libimage-exiftool-perl \
+  libgrokj2k1 \
+  grokj2k-tools \
+  adduser \
+  openjdk-17-jdk \
+  maven \
+  && rm -rf /var/lib/apt/lists/*
+
+# Install TurboJpegProcessor dependencies
+RUN mkdir -p /opt/libjpeg-turbo/lib
+COPY docker/Linux-JDK11/image_files/libjpeg-turbo/lib64 /opt/libjpeg-turbo/lib
```

Comment:
We might want to move the built jpeg-turbo library into a shared space in case we deprecate JDK11. Just a note